OpenPOS-APP/Exceptions/WriteToLogHandler.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenPOS_APP.Exceptions
+{
+    public class WriteToLogHandler
+    {
+        public static void WriteToLog(Exception e)
+        {
+            var path = AppDomain.CurrentDomain.BaseDirectory + "Logs";
+            Directory.CreateDirectory(path);
```

Comment:
Are you sure this will create a new file? And with a custom dynamic name for each exception that we want to log?